monai/apps/utils.py

```diff
@@ -34,9 +34,13 @@ def check_md5(filepath: str, md5_value: Optional[str] = None) -> bool:
     """
     if md5_value is not None:
         md5 = hashlib.md5()
-        with open(filepath, "rb") as f:
-            for chunk in iter(lambda: f.read(1024 * 1024), b""):
-                md5.update(chunk)
+        try:
+            with open(filepath, "rb") as f:
+                for chunk in iter(lambda: f.read(1024 * 1024), b""):
+                    md5.update(chunk)
+        except Exception as e:
+            print(f"Exception in check_md5: {e}")
```

Comment:
May I know why you catch the exception here? Maybe we can just throw the exception naturally?
Thanks.

Reply:
yes, this is a simple check function and it should behave in a consistent way. two options:
1. pass check return true otherwise return false
2. pass check return None otherwise raise an exception
either is good but mixing both is confusing.
i go for option 1 here so this simplifies the exception handling for the modules that call this function.